module/Finna/src/Finna/ILS/Driver/Mikromarc.php

```diff
@@ -1077,6 +1077,18 @@ public function getDefaultPickUpLocation($patron = false, $holdDetails = null)
      */
     public function getMyTransactionHistory($patron, $params)
     {
+        $history = [
+            'count' => 0,
+            'transactions' => []
+        ];
+        // Do not fetch loan history if userDisabled conditions are true
+        if (!empty($this->config['getMyTransactionHistory']['userDisabled'])
```

Comment:
I still think this should simply check `if (!($patron['loan_history'] ?? false))` without any configuration. So, if the variable is not available, default to false so that the history is never exposed unless we are sure it's enabled for the user.

Reply:
Oh, now I got it...

Reply:
And thats tweaked